README.md

```diff
@@ -1,66 +1,128 @@
 
-7. Read and accept the Sensory license agreement.
-Wait for the script to complete the installation. This can take some time.
+   ```./auto_install.sh xvf3100```
 
-8. As a final step, the script will open http://localhost:3000 in a browser on the Raspberry Pi. Enter your Amazon Developer credentials and close the browser window when prompted. (You won't have to do this if you already have a valid configuration file.)  
-If you see a `400 Bad Request - HTTP` error it may be necessary to add http:// URLs to your device origins and return fields. To do this, go to your amazon developer AVS home page (https://developer.amazon.com/avs/home.html) and from `My products` select `manage` and then `security profile`. Now add http://localhost:3000 to the `Allowed origins` and http://localhost:3000/authresponse to the `Allowed return URLs`. Now refresh the browser window with the original error.
+   for the xCORE VocalFusion XVF3100.
 
-9. Enter `sudo reboot` to reboot the Raspberry Pi and complete the installation.
+   Read and accept the AVS Device SDK license agreement.
+
+8. You will be asked whether you want the Sample App to run automatically when the Raspberry Pi boots. It is recommended that you respond "yes" to this option.
+
+9. Read and accept the Sensory license agreement. Wait for the script to complete the installation. The script is configuring the Raspberry Pi audio system, downloading and updating dependencies, building and configuring the AVS Device SDK. It takes around 30 minutes to complete.
+
+10. Enter `sudo reboot` to reboot the Raspberry Pi and complete the installation.
+
+11. If you selected the option to run the Sample App on boot you should now be able to complete the registration by following the steps from 2 onward here:
+https://github.com/alexa/avs-device-sdk/wiki/Raspberry-Pi-Quick-Start-Guide-with-Script#finish-authorization-using-login-with-amazon
+
+12. Now you can execute an AVS command such as "Alexa, what time is it?".
+
+   On the XMOS **xCORE VocalFusion XVF3510 Kit for Amazon AVS**, the LED on the Pi HAT board will change colour when the system hears the "Alexa" keyword, and will then cycle back and forth whilst waiting for a response from the Amazon AVS server.
+
+   On the XMOS **xCORE VocalFusion Stereo 4-Mic Kit for Amazon AVS**, the LEDs on the development board should reflect the approximate direction from which the microphones are receiving a stimulus.
 
 ## Running the AVS SDK Sample App
-The following commands will be available to run from a terminal:
+The automated installation script creates a number of aliases which can be used to execute the AVS Device SDK client, or run the unit tests:
 - `avsrun` to run the Sample App.
-- `avsauth` to re-authenticate your Alexa device details.
-- `avsunit` to run the unit tests.
-- `avssetup` to re-install the Sample App.
+- `avsunit` to run the unit tests (invoke Amazon's `test.sh`).
+- `avssetup` to re-install the Sample App (re-run XMOS modified `setup.sh`).
```

Comment:
I haven't run these aliases for some time... It is worth testing them. If they don't work we could remove them from this README.

Reply:
Ok, avsunit won't compile on the Pi until you remove the '-j2' from the invocation in test.sh (from xmos/sdk-device-setup). I'll see about raising a pull request on that repo. Once the tests compile, these are the results:
````
...
99% tests passed, 6 tests failed out of 1919

Label Time Summary:
slowtest     = 320.38 sec*proc (51 tests)
timertest    =  26.43 sec*proc (81 tests)

Total Test time (real) = 639.45 sec

The following tests did not run:
	  6 - AlertSchedulerTest.test_scheduleAlert (Disabled)
	  7 - AlertSchedulerTest.test_reloadAlertsFromDatabaseWithScheduling (Disabled)
	 52 - AlertTest.test_isPastDue (Disabled)
	155 - AudioPlayerTest.test_endOffset (Disabled)
	183 - BluetoothTest.test_handlePairDevices (Disabled)
	821 - SafeCTimeAccessTest.test_gmTimeMultithreadedAccess (Disabled)
	822 - SafeCTimeAccessTest.test_localtimeMultithreadedAccess (Disabled)
	828 - SharedDataStreamTest.test_readerRead (Disabled)
	838 - SharedDataStreamTest.testTimer_concurrencyNonblockableWriterDualReader (Disabled)
	898 - TaskThreadTest.test_moniker (Disabled)
	1775 - TimeZoneHandlerTest.settingCallbackFails (Disabled)
	1834 - */MediaPlayerTest.testSlow_unsteadyReads/* (Disabled)
	1835 - */MediaPlayerTest.testSlow_recoveryFromPausedReads/* (Disabled)
	1838 - */MediaPlayerTest.test_setOffsetOutsideBounds/* (Disabled)
	1839 - */MediaPlayerTest.test_setSourceResetsOffset/* (Disabled)

The following tests FAILED:
	1502 - SensoryKeywordTest.test_getExpectedNumberOfDetectionsInFourAlexasAudioFileForOneObserver (SEGFAULT)
	1503 - SensoryKeywordTest.test_getExpectedNumberOfDetectionsInFourAlexasAudioFileForTwoObservers (SEGFAULT)
	1504 - SensoryKeywordTest.test_getExpectedNumberOfDetectionsInAlexaStopAlexaJokeAudioFileForOneObserver (Child aborted)
	1506 - SensoryKeywordTest.test_getStreamClosedState (Child aborted)
	1507 - SensoryKeywordTest.test_getExpectedNumberOfDetectionsInAlexaStopAlexaJokeAudioFileWithRandomDataAtBeginning (Child aborted)
	1845 - */MediaPlayerTest.test_consecutiveSameApiCalls/* (Child aborted)
Errors while running CTest
make: *** [Makefile:130: test] Error 8
````
Do we keep them?

Reply:
avssetup no longer works, removing it...

Reply:
As discussed, we can remove the avsunit alias, since I didn't manage to run the test on the Alexa master branch of avs-device-sdk.